e2eshark/onnx/operators/reduceprod/model.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2024 Advanced Micro Devices
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# run.py creates runmodel.py by concatenating this file model.py
+# and tools/stubs/onnxmodel.py
+# Description: testing add
+# See https://onnx.ai/onnx/intro/python.html for intro on creating
+# onnx model using python onnx API
+import numpy, torch, sys
+import onnxruntime
+from onnx import numpy_helper, TensorProto, save_model
+from onnx.helper import make_model, make_node, make_graph, make_tensor_value_info
+from onnx.checker import check_model
+
+# import from e2eshark/tools to allow running in current dir, for run through
+# run.pl, commutils is symbolically linked to allow any rundir to work
+sys.path.insert(0, "../../../tools/stubs")
+from commonutils import E2ESHARK_CHECK_DEF
+
+# Create an instance of it for this test
+E2ESHARK_CHECK = dict(E2ESHARK_CHECK_DEF)
+
+
+# Create an input (ValueInfoProto)
+X = make_tensor_value_info("X", TensorProto.FLOAT, [2, 3, 4])
+# Y = make_tensor_value_info("Y", TensorProto.INT64, [3])
+
+# Create an output
+Z = make_tensor_value_info("Z", TensorProto.FLOAT, [1,1,1])
+
+# Create a node (NodeProto)
+reduceprodnode = make_node(
+    "ReduceProd", ["X"], ["Z"], "reduceprodnode"  # node name  # inputs  # outputs
+    # "ReduceProd", ["X", "Y"], ["Z"], "reduceprodnode"  # node name  # inputs  # outputs
```

Comment:
Can remove the commented out lines, was perhaps good for experimenting, but once you got test ready, good to have it clean